examples/DataFactory/DataFactory.Tests.ps1

```diff
@@ -70,61 +89,71 @@ Describe 'Verify Data Factory' {
 }
 
 Describe 'Verify Data Factory Linked Service' {
-  it 'Should contain a data factory with a linked service named BenchpressStorageLinkedService' {
+  BeforeAll {
+    $Script:linkedServiceName = 'BenchpressStorageLinkedService'
```

Comment:
Can this be moved up to the global `BeforeAll`?

Reply:
Since it's only being used in the linked service describe block, I want to keep it here.